our-team.php

```diff
@@ -3,180 +3,185 @@
 head('Our Team', true);
 ?>
 <main class="page-content">
-        <!-- Classic Breadcrumbs-->
-        <section class="breadcrumb-classic">
-          <div class="rd-parallax">
-            <div data-speed="0.25" data-type="media" data-url="images/breadcrumbs-parallax.jpg" class="rd-parallax-layer"></div>
-            <div data-speed="0" data-type="html" class="rd-parallax-layer section-top-75 section-md-top-150 section-lg-top-260">
-              <div class="shell">
-                <ul class="list-breadcrumb">
-                  <li><a href="/">Home</a></li>
-                  <li><a href="/about">About Us</a></li>
-                  <li>Our Team</li>
+  <!-- Classic Breadcrumbs-->
+  <section class="breadcrumb-classic">
+    <div class="rd-parallax">
+      <div data-speed="0.25" data-type="media" data-url="images/breadcrumbs-parallax.jpg" class="rd-parallax-layer"></div>
+      <div data-speed="0" data-type="html" class="rd-parallax-layer section-top-75 section-md-top-150 section-lg-top-260">
+        <div class="shell">
+          <ul class="list-breadcrumb">
+            <li><a href="/">Home</a></li>
+            <li><a href="/about">About Us</a></li>
+            <li>Our Team</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="section-75 section-md-100 section-lg-150">
+    <div class="shell range-offset-1">
+      <div class="range">
+        <div class="cell-lg-6">
+          <h1>Our Team</h1>
+          <h6>These people are the reason for our success and expertise.</h6>
+        </div>
+      </div>
+      <div class="range">
+        <div class="cell-lg-6 cell-sm-6 cell-md-12">
+          <div class="range range-sm-middle">
+            <div class="cell-md-6"><img src="images/bio-pics/juan-ruiz.jpg" alt="" width="360" height="404" class="img-responsive" />
+            </div>
+            <div class="cell-md-6">
+              <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
+                <h6 class="h6-with-small"><a href="#"> Juan Ruiz</a><span class="small text-silver-chalice">President</span></h6>

```

Comment:
Add a little comment for each person's section saying who it is so I don't have to dig in as far.